components/layouts/circles-page.tsx

```diff
@@ -61,7 +61,9 @@ export const CirclesPage: FC<CirclesPageProps> = ({ circles }) => {
           p="md"
           top={0}
           backgroundImage="/images/white_marble.png"
-          backgroundColor="Menu"
+          backgroundColor="#FFFFFF"
```

Comment:
```suggestion
          backgroundColor="white"
```

カラーコードでなくトークンを使用してください。

@nakata0047